packages/account/src/providers/chains.ts

```diff
@@ -4,7 +4,7 @@ export const CHAIN_IDS = {
     foundry: 31337,
   },
   fuel: {
-    beta5: 0,
-    devnet: 10,
+    devnet: 0,
+    testnet: 0,
```

Comment:
I'm sure I'm just out of context here, but what's the reasoning behind using identical chain IDs for `devnet` and `testnet`?

> cc @Torres-ssf @danielbate

Reply:
@petertonysmith94 Why it is worth dropping the asset for the devnet?